tsconfig.json

```diff
@@ -34,6 +34,7 @@
         "src/aiService.ts",
         "src/game.ts",
         "src/gameLogic.ts",
+        "src/common.ts",
```

Comment:
Also need to update here so that typescript compiler what files it needs to mess up with.